pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -114,10 +114,24 @@ func NewSnapshotManagerFromCluster(params map[string]interface{}, config map[str
 	}
 	logger.Infof("SnapshotManager: Get ivdPETM from the params map, VirtualCenter=%v, port=%v", params["VirtualCenter"], params["port"])
 
+	// TODO: Initialize PVC PETM.
+
+	// Initialize dummy s3 config.
+	s3Config := astrolabe.S3Config{
+		Port:      0,
+		Host:      nil,
+		AccessKey: "",
+		Secret:    "",
+		Prefix:    "",
+		URLBase:   "VOID_URL",
+	}
+
+	// Initialize the DirectProtectedEntityManager
+	depm := server.NewDirectProtectedEntityManager([]astrolabe.ProtectedEntityTypeManager{ivdPETM}, s3Config)
```

Comment:
NewDirectProtectedEntityManagerFromParamMap might be a better fit.

Reply:
ConfigInfo does not have exported fields, I cannot use that method as of yet without making astrolabe changes. Will take it up later sprints

Reply:
here is an example, https://github.com/lintongj/astrolabe/blob/dcd1231f13364fcbe677e3b09095cf9641fe2f06/pkg/pvc/tests/pvc_protected_entity_type_manager_test.go#L49

Reply:
`NewConfigInfo` is not available in astrolabe as of yet. I don't want to block this checkin while waiting for this.

Reply:
I see. that's part of Dave's PVC PE change.